src/components/Home/styles.js

```diff
@@ -0,0 +1,53 @@
+import { StyleSheet } from "react-native"
+import { labelWhiteColor, secondaryColor } from './../../common/includes'
+
+export default styles = StyleSheet.create({
+    container: {
+        padding: 15,
+
+        // flexDirection: 'row'
+    },
+    card1: {
+        flexDirection: 'row',
+        padding: 20,
+        backgroundColor: secondaryColor,
+        borderRadius: 10,
+        marginBottom: 20,
+        justifyContent: 'space-between'
+    },
+
+    cardLabelsTitle: {
+        fontSize: 40,
+        color: labelWhiteColor,
+        fontFamily: 'serif'
+    },
+    cardLabels: {
+        color: 'white',
+        marginTop: 10,
+        fontFamily: 'sans-serif',
+        fontSize: 18
+    },
+    image: {
+        marginTop: 60,
+        // marginRight: -20,
+        marginLeft: -25,
```

Comment:
Is this needed? Why is it a negative margin?

Reply:
negative margin is to make the image look aligned in the card. I will try to resolve it using another way and then push it again.